modules/test/playwright/tests/frontend-js-spa-web/utils/isSPAEnabled.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Page} from '@playwright/test';
+
+export default async function isSPAEnabled({page}: {page: Page}) {
+	await page.waitForTimeout(200);
+
+	const liferaySPAOutput = await page.evaluate(() => Liferay.SPA);
+
+	return liferaySPAOutput !== undefined;
```

Comment:
This is the main fix of the PR. The flakiness comes from the fact that `Liferay.SPA` is not immediately present in global namespace. `Liferay` exists immediately, but not `Liferay.SPA`. It appears for me locally after 10-20ms. I don't think there is any way to explicitly wait for this in PW, so I added an arbitrary wait of 200ms before the check.

Reply:
Maybe we could use the `.lfr-spa-loading` class that is is used while the application initialize.

This seems to work locally

```
	await expect(page.locator('.lfr-spa-loading')).not.toBeInViewport();
	// await page.waitForTimeout(200);
```

![image](https://github.com/user-attachments/assets/c1f49d50-f92b-4c0c-8c8e-e5816c72c645)

Reply:
@juanjofgliferay Yeah, that works perfectly :ok_hand: 

I pushed the change.